test/extended/syncer/syncer_utils.go

```diff
@@ -170,5 +170,9 @@ func (s *SyncTarget) Delete(k *exutil.CLI) {
 
 // Clean the SyncTarget resource
 func (s *SyncTarget) Clean(k *exutil.CLI) error {
+	// TODO: Temp solution for known limitation: https://github.com/kcp-dev/kcp/issues/999
+	// Need to remove after the issue solved
+	mySyncerKey, _ := k.WithoutNamespace().WithoutKubeconf().WithoutWorkSpaceServer().Run("get").Args("--server="+s.WorkSpaceServer, "synctarget", s.Name, `-o=jsonpath={.metadata.labels.internal\.workload\.kcp\.dev/key}`).Output()
+	k.AsPClusterKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("delete").Args("ns", "-l", "internal.workload.kcp.dev/cluster="+mySyncerKey).Execute()
```

Comment:
Do we need check err?

Reply:
IMO, we don't need to check the results of the pCluster left ns clean up results as it's just a temp solution not related the function, when DEV finished the `kubectl kcp workload unsync` function we could change the logic.